### ISSUE TITLE
HPN-SSH version

### ISSUE THREAD

**Issue by asimoneo** [billziss-gh/sshfs-win#85]
HPN-SSH is a set of pathes to allow lowering or almost liquidating cpu bottleneck during intense ssh file transfers, which is currently cutting my throughtput speed down 3 times or so.

I am using encfs to work with my files, which stores encrypted versions on mounted via sshfs-win remote hdd partition from linux server, so double encryption is totally not necessary here.

Can we have an option to have hpn-ssh?

**Comment by billziss-gh** [comment]
HPN-SSH looks very interesting.

Have you tried the `-o ssh_command=...` option to sshfs from the command line?

**Comment by asimoneo** [comment]
nope, I did not. I am not that familiar with techy stuff, but feel free to contact me for beta testing and so on. Hopefully I can assist and we could create a small gui for that, btw take a look at rclone project and it's android variation - magisk module for rclone: https://github.com/Magisk-Modules-Repo/com.piyushgarg.rclone

**Comment by its0ka** [comment]
@billziss-gh nothing new? I tried replacing ssh.exe with other versions and no success

**Comment by its0ka** [comment]
i lose too much speed because my mini pc can't handle encryption, would like to use HPN ssh with none cipher
